chain-impl-mockchain/src/vote/tally.rs

```diff
@@ -98,15 +90,11 @@ impl Tally {
         }
     }
 
-    pub fn private_encrypted(&self) -> Result<(&EncryptedTally, &Value), TallyError> {
+    pub fn private_encrypted(&self) -> Result<(&EncryptedTally, Value), TallyError> {
```

Comment:
is the stake return value used somewhere? I don't think so, as we don't have such thing for the public tally

Reply:
It's used in `ProposalManager` somewhere, I guess it could probably just return a `Result<&EncryptedTally>`